server/sonar-server/src/main/java/org/sonar/server/ui/ws/GlobalAction.java

```diff
@@ -122,7 +125,27 @@ private void writeQualifiers(JsonWriter json) {
   }
 
   private void writeVersion(JsonWriter json) {
-    json.prop("version", server.getVersion());
+    String technicalVersion = server.getVersion();
+    String functionalVersion = technicalVersion;
```

Comment:
I've updated, I'm not sure the code is clearer because I split the functional version